### DISCUSSION TITLE
Grouping flatpak-system and flatpak-user into one in config

### DISCUSSION THREAD

**Discussion by mateowoetam** [fastfetch-cli/fastfetch#1019] [General]
I don't know if this is possible, but I would like to know if there is any way to setup the packages part in the fastfetch `config.jsonc` file that allows me to have both `(flatpak-system)`, and  `(flatpak-user)` together into just `(flatpak)` I thought it would be with a `"format"` config but haven't found one that works.

My config looks like this right now:
```
        {
            "type": "packages",
            "key": "Packages",
            "keyColor": "blue"
        },
```

**Comment by CarterLi** [comment]
`"format": "{flatpak-all}"`

See `fastfetch -h packages-format` for detail
